src/Inquirer/Prompts/ListInputMessage.cs

```diff
@@ -0,0 +1,19 @@
+﻿using System;
+
+namespace InquirerCore.Prompts
+{
+    public class ListInputMessage
```

Comment:
Instead of creating a ListInputMessage, it can be a ConsoleMessage or just a Message to create a more generic class that will allow us to use in more places.
It can just have now the string message and the color, and the logic related to the color can be outside the class.

Reply:
Obrigado pelo feedback! Você tem razão. Uma classe mais genérica permitiria o uso em mais lugares e não exclusivamente no ListInput. Estarei revendo essa mudança no próximo PR.

Reply:
Thanks for the feedback! You're right. A more generic class would allow use in more places and not exclusively on ListInput. I will be reviewing this change in the next PR.